interactions/four_particle/cpp/integral.cpp

```diff
@@ -320,12 +311,11 @@ dbl integrand_full(
 
     switch (integral_kind) {
         case CollisionIntegralKind::F_1_vacuum_decay:
-             return temp * F_1_vacuum_decay();
+             return temp * F_1_vacuum_decay(f);
         case CollisionIntegralKind::F_f_vacuum_decay:
             return temp * F_f_vacuum_decay();
         case CollisionIntegralKind::Full_vacuum_decay:
-            return temp * (F_1_vacuum_decay() + f[0] * F_f_vacuum_decay());
-
+            return temp * (F_1_vacuum_decay(f) + f[0] * F_f_vacuum_decay());
```

Comment:
Make the function signatures consistent please

Reply:
Will do.